kasparovsyncd/sync.go

```diff
@@ -211,16 +211,36 @@ func fetchBlock(client *jsonrpc.Client, blockHash *daghash.Hash) (
 // addBlocks inserts data in the given rawBlocks and verboseBlocks pairwise
 // into the database. See addBlock for further details.
 func addBlocks(client *jsonrpc.Client, rawBlocks []string, verboseBlocks []rpcmodel.GetBlockVerboseResult) error {
+	db, err := database.DB()
+	if err != nil {
+		return err
+	}
+
+	blocks := make([]*rawAndVerboseBlock, 0)
+	blocksMap := make(map[string]*rawAndVerboseBlock)
```

Comment:
Please mention what does it map to what.
Preferably in name, but if too long, than at least a comment